app/domain/package.scala

```diff
@@ -14,16 +14,15 @@
  * limitations under the License.
  */
 
-import org.joda.time._
 import play.api.libs.json._
 import uk.gov.hmrc.auth.core.retrieve.{GatewayInformation, ItmpAddress, ItmpName, MdtpInformation}
 import uk.gov.hmrc.auth.core.{Enrolment, EnrolmentIdentifier}
+import java.time.LocalDate
 
 package object domain {
-  implicit val dFormat: Format[LocalDate] = new Format[LocalDate] {
-    override def reads(json: JsValue): JsResult[LocalDate] = JodaReads.DefaultJodaLocalDateReads.reads(json)
-    override def writes(o: LocalDate): JsValue = JodaWrites.DefaultJodaLocalDateWrites.writes(o)
-  }
+
+  implicit val reads: Reads[LocalDate] = play.api.libs.json.Reads.DefaultLocalDateReads
```

Comment:
as play json is already aware of LocalDate Reads you probably don't need to explicitly specify these here? (worth trying without)